tensorrt_llm/_torch/speculative/mtp.py

```
@@ -1041,17 +1084,37 @@ def draft_sampler(
+        else:
+            # Simple argmax if no TP or no model config
+            draft_tokens = torch.argmax(logits, dim=-1).type(torch.int32)
 
-        draft_tokens = torch.argmax(logits, dim=-1).type(torch.int32)
         return draft_tokens
 
 
 class MTPEagleWorker(MTPWorker):
 
-    def __init__(self, spec_config: "MTPDecodingConfig"):
+    def __init__(self, spec_config: "MTPDecodingConfig",
```

Ruff (0.12.2)

Undefined name `MTPDecodingConfig`

(F821)